src/components/main/flight/ReceiptWithFlightScreen.js

```diff
@@ -1,232 +1,257 @@
-              {/*Years to neutralize*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('YEARS TO COMPENSATE')}</Text>
-              <Text style={styles.receiptTextRight}>{years}</Text>
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Cost of transaction*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('PRICE')}</Text>
-              <Text style={styles.receiptTextRight}>${total_cost}</Text>
-            </View>
+          <View style={styles.bottomInnerView}>
+            <List>
+              {/* List Item 1 */}
+              <ListItem>
+                <Left>
+                  <Text style={styles.itemTitle}>Flight Distance (km)</Text>
+                </Left>
+                <Right>
+                  <Text style={styles.itemValue}>{distance}</Text>
+                </Right>
+              </ListItem>
+              {/* List Item 2 */}
+              <ListItem>
+                <Left>
+                  <Text style={styles.itemTitle}>
+                    Tons of CO
+                    <Text
+                      style={{
+                        fontSize: moderateScale(7, 0.125),
+                        lineHeight: 1,
+                      }}>
+                      2
+                    </Text>
+                  </Text>
+                </Left>
+                <Right>
+                  <Text style={styles.itemValue}>{footprint}</Text>
+                </Right>
+              </ListItem>
+              {/* List Item 3 */}
+              <ListItem>
+                <Left>
+                  <Text style={styles.itemTitle}>Planted Trees</Text>
+                </Left>
+                <Right>
+                  <Text style={styles.itemValue}>{treeNum}</Text>
+                </Right>
+              </ListItem>
+              {/* List Item 4 */}
+              <ListItem>
+                <Left>
+                  <Text style={styles.itemTitle}>Years to Compensate</Text>
+                </Left>
+                <Right>
+                  <Text style={styles.itemValue}>{years}</Text>
+                </Right>
+              </ListItem>
+              {/* List Item 5 */}
+              <ListItem>
+                <Left>
+                  <Text style={styles.itemTitle}>Donation Amount</Text>
+                </Left>
+                <Right>
+                  <Text style={styles.itemValue}>${total_cost}</Text>
+                </Right>
+              </ListItem>
+            </List>
+            {/* SignUp Button */}
+            <TouchableOpacity
+              onPress={() => this.handleStripePayment()}
+              disabled={this.state.isLoading}
+              style={styles.submitButton}>
+              {/* Not Loading Hide ActivityIndicator */}
+              {!this.state.isLoading && (
+                <Text style={styles.submitLabel}>Pay With Stripe</Text>
```

Comment:
Either add in an `isLoading` prop or remove this check and the ActivityIndicator all together. You're passing in a null prop which luckily defaults to isLoading = false in this case.

Also, an activity indicator will never be shown, since it immediately redirects to our Payment Loading screen